clients/imodels-client-management/src/base/interfaces/apiEntities/ChangesetInterfaces.ts

```diff
@@ -21,52 +25,97 @@ export enum ContainingChanges {
   GlobalProperties  = 1 << 5
 }
 
+/** Application information. */
 export interface Application {
+  /** Application name. */
   name: string;
 }
 
+/** Synchronization information. */
 export interface SynchronizationInfo {
+  /** List of files that were processed by the synchronization. */
   changedFiles: string[];
 }
 
+/** DTO for links that belong to Changeset entity returned from iModels API. */
 export interface ChangesetLinks {
+  /** Link to upload the Changeset file. Points to file storage. */
   upload: Link;
+  /** Link to download the Changeset file. Points to file storage. */
```

Comment:
Link from where to download the Changeset file. Link points to a remote storage.

Reply:
Fixed: https://github.com/iTwin/imodels-clients/blob/1aac582dbc6b1ead914650a280cabe53a4f0487e/clients/imodels-client-management/src/base/interfaces/apiEntities/ChangesetInterfaces.ts#L45